src/Commands/CreateTenant.php

```diff
@@ -93,7 +93,8 @@ public function handle()
             return;
         }
 
-        $tenant = new \Slides\Saml2\Models\Tenant([
+        $class = config('saml2.tenantModel');
```

Comment:
Please add a default value, otherwise it will break for people who already have that config.

Reply:
I think Laravel does a merge between the application config and the package one, but I will set that up.